server/rpc/httphealth/httphealth.go

```diff
@@ -34,36 +34,34 @@ type CheckResponse struct {
 
 // NewHandler creates a new HTTP handler for health checks.
 func NewHandler(checker grpchealth.Checker) (string, http.Handler) {
+	const serviceName = "/yorkie.v1.YorkieService/health"
```

Comment:
I think now we have introduced `HealthV1ServiceName` constant, we can remove this variable.

Reply:
It preserves the structure of the existing grpchealth, which we don't need! Thanks:)